src/command_modules/azure-cli-servicefabric/azure/cli/command_modules/servicefabric/custom.py

```diff
@@ -266,8 +266,10 @@ def new_cluster(client,
     validate_result = _deploy_arm_template_core(
         resource_group_name, template, parameters, deployment_name, 'incremental', True)
     if validate_result.error is not None:
-        raise CLIError("Template validates error \n'{}'".format(
-            validate_result.error))
+        errors_detailed = _build_detailed_error(validate_result.error, [])
+        error_message = "Error validating template. See below for more information. \n"
+        error_message += "".join(errors_detailed)
```

Comment:
Would prefer not to have the `\n` manually entered for each line yourself then add the new lines as part of the join.
```
raise CLIError('\n'.join(error_messages))
```

Reply:
Thanks, will do